claims: use binascii for head imprint

azul opened the pull request:
Also first pair of useful tests.

azul committed:
claims: use binascii for head imprint

Also first pair of useful tests.

azul committed:
travis: run tests with plugins

bogdan-kulynych commented:
Why don't you want to use `bytes2ascii` from `claimchain.utils`? It's base58 encoding, so more suitable for out-of-band transmission.

azul commented:
@bogdan-kulynych oh... did not know it existed. Thanks for pointing it out!

azul committed:
use bytes2ascii from claimchain.tools

It is base58 encoding, so more suitable for out-of-band transmission.
Thanks @bogdan-kulynych for pointing it out.
